python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2079,7 +2090,15 @@ def test_parquet_allnull_str(tmpdir, engine):
     )
 
     pdf.to_parquet(pandas_path)
-    actual_gdf = cudf.read_parquet(pandas_path, engine=engine)
+    if engine == "pyarrow":
+        with pytest.warns(
+            UserWarning,
+            match="Using CPU via PyArrow to read Parquet dataset. This option "
+            "is both inefficient and unstable!",
+        ):
+            actual_gdf = cudf.read_parquet(pandas_path, engine=engine)
+    else:
+        actual_gdf = cudf.read_parquet(pandas_path, engine=engine)
```

Comment:
With above suggestion:
```suggestion
    with _hide_pyarrow_parquet_cpu_warnings(engine):
        actual_gdf = cudf.read_parquet(pandas_path, engine=engine)
```